src/main/java/utils/EnumsForSprites.java

```diff
@@ -1,3 +1,3 @@
 package utils;
 
-public enum EnumsForSprites {ISTRANSVERSABLE, NOTTRANSVERSABLE, PLAYER, ALLIGATOR, ALLIGATORDEN, GOAL}
+public enum EnumsForSprites {ISTRANSVERSABLE, NOTTRANSVERSABLE, PLAYER, ALLIGATOR, ALLIGATORDEN, GOAL, Chaser}
```

Comment:
Might as well make chaser capitalized like CHASER, to follow suit

Reply:
Yes, good call. Will fix.